docs/deployment/kubernetes.md

```diff
@@ -82,10 +66,29 @@ helm install <deployment name> quickwit/quickwit -f values.yaml
 
 6. Check that Quickwit is running
 
-It might take some time for the cluster to start. During the startup process individual pods might restart themselves. The command that you typed on previous step should print out instructions on how to set port forwarding to connect to the cluster. While this command is running, you can access the cluster UI by navigating to http://127.0.0.1:8080 in your browser. You can also check the version of Quickwit by running:
+It might take some time for the cluster to start. During the startup process individual pods might restart themselves several times. The command on the previous step will print the instructions on how to connect to the cluster. This endpoint can be used to access the quickwit search UI, as well execute standard API commands against.
 
-```bash
-curl http://127.0.0.1:8080/api/v1/version
+## Using PostgreSQL as a metadata store
+
+The file-backed metastore is mainly useful for testing purposes. Though a file-backed metastore might be easier to setup, we strongly encourage you to use a PostgreSQL metastore in production. For the quickwit installation to work with PostgreSQL metadata you need to provide connection PostgreSQL information instead of metastore url:
```

Comment:
```suggestion
The file-backed metastore is mainly useful for testing purposes. Though a file-backed metastore might be easier to setup, we strongly encourage you to use a PostgreSQL metastore in production. For the quickwit installation to work with PostgreSQL metadata you need to provide connection PostgreSQL information instead of metastore URI:
```